decidim-admin/config/locales/ca.yml

```diff
@@ -636,4 +636,4 @@ ca:
   time:
     formats:
       datepicker: "%m/%d/%Y"
-      timepicker: "%d/%m/%Y %H:%M"
\ No newline at end of file
+      timepicker: "%d/%m/%Y %H:%M"
```

Comment:
Yeah, in the spec the talk about "quote styles": http://www.yaml.org/spec/1.2/spec.html#id2786942, nothing compulsory or even recommended.

I'm fine with using single quotes for YAML files when possible (although I'd prefer to be able to choose). What really feels weird is that they choose a "no final newline" style, since that's clearly minoritary.